app/src/main/res/layout/fragment_audio_player.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/audioRecordLayout"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    tools:context=".main_screen.audio_player.AudioPlayerFragment">
+
+    <View
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        android:layout_alignParentBottom="true"
+        android:background="?android:attr/listDivider" />
+
+    <RelativeLayout
+        android:id="@+id/audio_play_layout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content" >
+
+        <ImageView

```

Comment:
Might be a good idea to add a contentDescription for the imageView

Reply:
It's just a white background though, so that the background of the profile page cannot be seen through

Reply:
Mb, I just followed Cirrus indication xD